client/src/Features/MicrosoftLearn/MicrosoftLearnFilterComponentProps.ts

```diff
@@ -0,0 +1,35 @@
+import { Product, LearnType, Level, Role } from '../../Models/Learn';
+import { FilterType } from '../../Models/Learn/FilterType.model';
+import { FormEvent } from 'react';
+import { FilterComponentStyles } from './MicrosoftLearnFilterComponentUtils';
+
+export type LearnTypeName = 'Learning Path' | 'Module';
+
+export type LearnTypeFilterOption = {
```

Comment:
any specific reason for creating this?

Reply:
The existing `LearnType.model` (~ `LearnTypeDto`), unlike the other three (product, role, level), is a simple string, and not an object with the properties {`id`, `name`}. The same (`LearnTypeFilterOption`) was defined here to make the rest of the implementation of `FilterComponents`/`FilterItems` common for all four.